### PR TITLE
CT-1782 fix permissions issues

### PR DESCRIPTION
**Pull request by zajca**
Jira: CT-1782
Connection PR: https://github.com/keboola/connection/pull/5342

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

### CODE REVIEW

**Comment by vojtabiberle** [comment]
Padá mi tento test: https://github.com/keboola/kbc-manage-api-php-client/blob/a28507a88377a2cd729da075ec7f778e5391c8f3/tests/ProjectWithProtectedDefaultBranchTest.php#L355

**Review by vojtabiberle — CHANGES_REQUESTED** [review]

File: tests/ProjectWithProtectedDefaultBranchTest.php [hunk 1]
```diff
@@ -453,6 +453,20 @@ public function testAddFeatureToProject(): void
...
         }
+
+        // test is super admin can add feature
+        $this->client->addProjectFeature($projectId, $featureName);
+        $this->client->removeProjectFeature($projectId, $featureName);
+
+        try {
+            $this->client->removeUserFromProject($projectId, $this->superAdmin['id']);
+        } catch (ClientException $e) {
+            // ignore failure
```
[the remark this answers is not in the document]
Comment (zajca) [reply tests/ProjectWithProtectedDefaultBranchTest.php]: odstranil sem to radši https://github.com/keboola/kbc-manage-api-php-client/pull/276/commits/5946fc70aef972160d962b29096a7e08da0a1910

**Comment by zajca** [comment]
ten test mě nepadá, zkusil sem na masteru aj v PR
image

File: tests/ProjectWithProtectedDefaultBranchTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (vojtabiberle) [reply tests/ProjectWithProtectedDefaultBranchTest.php]: ok

**Comment by vojtabiberle** [comment]
> ten test mě nepadá, zkusil sem na masteru aj v PR image

Tak to budeme považovat za nějakou haluz u mě.

**Review by vojtabiberle — APPROVED** [review]

**Pull request merged**